jpegtran/jpegtran_build.py

```diff
@@ -16,9 +16,10 @@
 ffi.set_source(
     "_jpegtran", SOURCE,
     sources=["src/epeg.c"],
-    include_dirs=["src"],
+    include_dirs=["src", "/opt/libjpeg-turbo/include"],
     define_macros=[("HAVE_UNSIGNED_CHAR", "1")],
-    libraries=["jpeg", "turbojpeg"])
+    libraries=["jpeg", "turbojpeg"],
+    library_dirs=["/opt/libjpeg-turbo/lib64"])
```

Comment:
These `/opt/` paths don't look very portable...  Why is this change needed?

Reply:
I didn't notice the hard code. I will fix it.

Reply:
At the time I added the hardcode, the features of libjpeg-turbo were only available in the dev branch. So I had to manually build a working environment. 

So far, I'm still not very sure how to pass the include dir and lib dir to the build script in a more elegant way, i.e. by  command line options, if the header and .so files are not in the default system paths, i.e. /usr/include or /usr/lib.

I try to pass it through environment vars `JPEG_TURBO_INCLUDE=/opt/libjpeg-turbo/include/ JPEG_TURBO_LIB=/opt/libjpeg-turbo/lib64`. And I try to give them default values as `/opt/...`. It seems that libjpeg-turbo is installed in `/opt` by default.